### PR TITLE
Change the invoker to DaemonSet

### PR DESCRIPTION
**Pull request by DanLavine**
This is a WIP for changing Invokers to a DaemonSet. It relies on this PR https://github.com/apache/incubator-openwhisk/pull/2689 to be merged first.

### CODE REVIEW

File: kubernetes/invoker/invoker.yml [hunk 1]
```diff
@@ -116,6 +122,12 @@ spec:
             value: "test_whisks"
           - name: "DB_WHISK_AUTHS"
             value: "test_subjects"
+
+          # UUID for the pod can be the hostname of the Kube node
+          - name: "INVOKER_ASSIGNED_UUID"
```
Comment (dgrove-oss) [inline kubernetes/invoker/invoker.yml]: Based on feedback, I changed the envvar in the PR to INVOKER_ASSIGNED_NAME

**Comment by csantanapr** [comment]
@DanLavine @dgrove-oss PR https://github.com/apache/incubator-openwhisk/pull/2689 is now merged

**Comment by dgrove-oss** [comment]
One final tweak needed;  the final PR that was merged used INVOKER_NAME (not INVOKER_ASSIGNED_NAME) for the envvar.

File: kubernetes/invoker/invoker.yml [hunk 2]
```diff
@@ -127,6 +136,12 @@ spec:
             value: "test_whisks"
           - name: "DB_WHISK_AUTHS"
             value: "test_subjects"
+
+          # UUID for the pod can be the hostname of the Kube node
+          - name: "INVOKER_ASSIGNED_NAME"
```
Comment (dgrove-oss) [inline kubernetes/invoker/invoker.yml]: Needs to be changed to "INVOKER_NAME"

**Comment by dgrove-oss** [comment]
Hi Dan.  I've created an updated version of this (#107).  I hope to get it merged ASAP (it is working nicely for me on both single-node and multi-node clusters).  I think you can go ahead and close this PR as being superseded by 107.

**Pull request closed without merging**